frontend/development-configurations/build-test-pa11y.sh

```diff
@@ -1,5 +1,10 @@
 yarn run build
-yarn run serve-static-files &
-yarn run pa11y &
-wait
-pkill -P $$
+yarn run serve-static-files & yarn run pa11y
+if [[ $pa11yreturncode = 0 ]]
```

Comment:
Looks like a copy paste error, this variable doesn't exist. I usually add these options to every shell script to catch errors like this.
```
set -o errexit
set -o pipefail
set -o nounset # this one will error when encountering an undefined variable
```

Instead of doing the `if $? = 0`, you could just

    exec yarn run pa11y

And the script will just exit with yarn's exit code. I don't know that you really need to print success/failure.